policykit/policykit/settings.py

```diff
@@ -282,13 +280,22 @@
         "file": {
             "class": "logging.FileHandler",
             "filename": env("LOG_FILE"),
-            "formatter": "file",
+            "formatter": "json",
         },
         "console": {"class": "logging.StreamHandler", "formatter": "console"},
     },
-    'loggers': loggers
+    'loggers': {
+        # Database logger for policy evaluation logs
+        "db": {"handlers": ["db_log"], "level": DEFAULT_LOG_LEVEL, "propagate": True},
+        # Only log DDTRACE warnings
+        "ddtrace": {"level": "WARNING", "propagate": True},
+        # Info for everything else
+        "": {"handlers": ["console", "file"], "level": LOG_LEVEL}
```

Comment:
I changed the logging so everything (besides ddtrace) just is logged to the file and to the console. I keep the `db_log` logging though as well, so that some logs are added to the database.